src/cartridges/int_adyen_controllers/cartridge/controllers/Adyen.js

```diff
@@ -408,7 +408,7 @@ function getPaymentMethods(cart, customer) {
     adyenConnectedTerminals: connectedTerminals,
     ImagePath: adyenURL,
     AdyenDescriptions: paymentMethodDescriptions,
-    amount: { value: paymentAmount, currency: currency },
+    amount: { value: parseFloat(paymentAmount), currency: currency },
```

Comment:
Where is this amount used? In case it's is used for the frontend components as well as a config field you need to use minor units.